api/features/views.py

```diff
@@ -752,3 +769,94 @@ def create_segment_override(
     serializer.is_valid(raise_exception=True)
     serializer.save(environment=environment, feature=feature)
     return Response(serializer.data, status=201)
+
+
+@swagger_auto_schema(
+    method="POST",
+    request_body=CreateFeatureExportSerializer(),
+    responses={201: CreateFeatureExportSerializer()},
+)
+@api_view(["POST"])
+@permission_classes([CreateFeatureExportPermissions])
+def create_feature_export(request: Request) -> Response:
+    serializer = CreateFeatureExportSerializer(data=request.data)
+    serializer.is_valid(raise_exception=True)
+
+    export_features_for_environment.delay(
+        kwargs={
+            "environment_id": serializer.validated_data["environment_id"],
+            "tag_ids": serializer.validated_data["tag_ids"],
+        }
+    )
+
+    return Response(serializer.validated_data, status=201)
+
+
+@swagger_auto_schema(
+    method="POST",
+    responses={201: FeatureImportSerializer()},
+)
```

Comment:
Can we use a serializer with a `serializers.FileField()` here?

Reply:
I was able to get this working with the import, but I'm less sure of the export. Either way, a small win.